LocalSettings.php

```diff
@@ -1423,7 +1423,21 @@
 			'tlh',
 			'zombie',
 		],
+		'+hispanowiki' => [
+			'w',
+			'v',
+			'n',
+			'u',
+			'd',
+			'c',
+			'commons',
+		],
 	],
```

Comment:
This needs to go after the ucroniaswiki block

Reply:
done